src/ZkFold/Wallet/Server/Types.hs

```diff
@@ -0,0 +1,23 @@
+module ZkFold.Wallet.Server.Types where
+
+import           Data.Text           (Text)
+import           Servant
+
+import           ZkFold.Wallet.Types
+
+type WalletAPI =
```

Comment:
This is actually CIP30 Wallet API. We will need to implement these functions to complete M2. "Symbolic Wallet API" gives some additional functions that we can complete later.

Reply:
Oh, so basic logic from [this issue](https://github.com/zkFold/smart-contract-wallet/issues/3) is actually all these functions? Gotcha